sensor_msgs/Image image1
sensor_msgs/Image image2